data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/controller/MultipartController.java

```diff
@@ -171,6 +201,30 @@ private FormDataMultiPart createFormDataMultiPart(Object header, Object payload)
 
         return multiPart;
     }
+    
+    /**
+     * Retrieves an identity token and adds it to the given message.
+     *
+     * @param header the message.
+     */
+    private void addTokenToResponseHeader(Message header) {
```

Comment:
Instead of having this `void` method, it could return the `DynamicAttributeToken`, that will be set as security token on the header by the callers (it will avoid the side effect)